### ISSUE TITLE
Chromium 116 slow on older RevPis

### ISSUE THREAD

**Issue by steima** [variocube/image#3]
Example: kitchen showroom001.variocube.local

The user interface (Chromium on Xorg) responds very slowly.
CPU load is small.

Hardware:
Raspberry Pi Compute Module 3 Rev 1.0

**Comment by mlyngvo** [comment]
It probably has something to do with chromium and debian version, the latest image of VariocubeOS contains

`Chromium 112.0.5615.138 built on Debian 11.6, running on Debian 11.7`

while on the kitchen cube it is

`Chromium 116.0.5845.180 built on Debian 11.7, running on Debian 11.8`

We had the same problem on **Friday the 13th** (👹) last week when Mo / Christina tried to provision a cube for a customer and "accidentally" upgrade the debian and chromium version along the way. After re-writing the variocube OS image on the Pi and carefully install the variocube packages (no os upgrade this time), the UI works normally again.

**Comment by peterkarrer** [comment]
I have seen there are already newer version of Chromium available on Debian. Let's try to upgrade to 117 and see if the problem persists before any lengthy investigation.

**Comment by mlyngvo** [comment]
@peterkarrer Upgrading chromium to 117 seems to fix the issue in Debian 11.8.

The new version `117.0.5938.149`, however, is still in `debian-security` source. Until they push it stable, add this line to `/etc/apt/sources.list` in order to fetch the package.

```
deb http://deb.debian.org/debian-security/ bullseye-security main contrib non-free
```
